app/backend/wells/serializers_v2.py

```diff
@@ -376,3 +376,16 @@ class WellExportAdminSerializerV2(WellExportSerializerV2):
     class Meta:
         model = Well
         fields = WellListAdminSerializerV2.Meta.fields
+
+
+class WellAquiferSerializerV2(serializers.ModelSerializer):
+    """Serializes aquifer info for a selection of wells"""
+    aquifer_subtype = serializers.ReadOnlyField(source='aquifer.subtype.description')
+
+    class Meta:
+        model = Well
+        fields = (
+            "well_tag_number",
+            "aquifer",
+            "aquifer_subtype"
```

Comment:
With only these three params this API is very specific to Wally's needs. Should it be expanded to include all the fields of an aquifer so it is a generic as it is a public API?